### ISSUE TITLE
[Crash]: FTB Endeavour 1.8.0 has crashed!

### ISSUE THREAD

**Issue by xJoKeStAx** [FTBTeam/FTB-Mods-Issues#677]
### Mod

FTB

### Mod version

1.8.0

### Forge / Fabric version

202212152008-a5a6712906-release

### Modpack & version

FTB Endeavour 1.8.0

### What issue are you having?

An Error Occurred Whilst Launching

Failed to prepare instance 'FTB Endeavour'(40907201-b387-4896-gde4-5421024b6898). because.. java.io.IOException: Failed to execute asset update task.
![FTB Endeavour 1 8 0 crashed](https://user-images.githubusercontent.com/124860466/217726949-cd6d8d6e-4d76-4f6d-8e8b-216fa25a2874.png)

### Crashlogs

An Error Occurred Whilst Launching

Failed to prepare instance 'FTB Endeavour'(40907201-b387-4896-gde4-5421024b6898). because.. java.io.IOException: Failed to execute asset update task.

### Steps to reproduce

I click play and it attempts to load but then crashes

**Comment by MichaelHillcox** [comment]
App issue, not a mod issue, open issue can be found here https://github.com/FTBTeam/FTB-App/issues/832. 

TLDR, we're working on it. Fix should be released early next week
